{{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/app/apps.py

```diff
@@ -0,0 +1,6 @@
+from django.apps import AppConfig
+
+class MainAppConfig(AppConfig):
```

Comment:
I want to de-emphasize (and probably eventually remove) ``app``. You can call this `AppConfig` so long as your alias the import:
```python
from django.apps import AppConfig as DjangoAppConfig

class AppConfig(DjangoAppConfig):
  ...
```

I actually think we should rename this to `home` pursuant to @gone suggestion.